graphql.go

```diff
@@ -53,6 +55,12 @@ func ParseSchema(schemaString string, resolver interface{}, opts ...SchemaOpt) (
 func MustParseSchema(schemaString string, resolver interface{}, opts ...SchemaOpt) *Schema {
 	s, err := ParseSchema(schemaString, resolver, opts...)
 	if err != nil {
+		lines := strings.Split(schemaString, "\n")
+		for i, line := range lines {
+			lines[i] = fmt.Sprintf("%d: %v", i+1, line)
```

Comment:
I don't think adding this is good for all users of this pkg. It is certainly helpful and useful for you. But, in principle, a library should not logs, prints, or do anything with side effects if the API is not a side effect API.

Reply:
A better logging should be taken over by the user/caller of this function.

Reply:
A more elegant solution is not to use this function. Use `ParseSchema` directly and wrap it, like
```go
func YourOwnMustParseSchema(...) {
  err := ParseSchema(...)
  // do whatever you want with the error
}
```

Reply:
I also don't recommend the use of `MustXXX` function because they panic by convention.

Consider
```
err := ParseSchema(...)
if err != nil {
  logYourErr(err)
  os.Exit(DefineASpecificExitValue)
}
```
This is especially useful when your program might have multiple exit reason and you can interact with exit code in shell.

Reply:
ive the opposite opnions, if the library does not output sufficient debug messages, it will sway people away from using it. can you define "side effect API" ?

Reply:
By "side effect API", I mean an API which is designed to have side effects such as logging, networking, any kinds of IO, modifying of states of objects.

Back to this PR, we have 2 discussions here:
1. Should `MustParse` have side effects?
No, I think it should not because side effects have many more undesired consequences in general, even if it's simple as logging.
For example, my server may wants to redirect loggings to `stderr` and `stdout` to a network address and a logging database (elastic search for example), injecting a logger is the best solution. But, that will change the signature of this function, which breaks everybody's code.

2. Is a logging here even useful for most people?
I don't know. But I never need to log here.

If you truly want to provide a more descriptive msg, consider change the `Error() string` method of its returned error.

Reply:
okay added log string into error for panic